python/robot_properties_solo/quadruped12wrapper.py

```diff
@@ -75,6 +75,11 @@ def forward_robot(self, q=None, dq=None):
         self.pin_robot.framesForwardKinematics(q)
         self.pin_robot.centroidalMomentum(q, dq)
 
+    def start_recording(self, file_name):
+        p.startStateLogging(p.STATE_LOGGING_VIDEO_MP4, file_name + ".mp4")
+
+    def stop_recording(self, file_name):
+        p.stopStateLogging(p.STATE_LOGGING_VIDEO_MP4, file_name + ".mp4")
```

Comment:
Usually a filename includes the extension of the file. I would therefore remove the concatenation of `+ ".mp4"` and let the user provide the file with the right extension.

You could make the `file_name` in `stop_recording` optional. If no value is provided, stop the last `file_name` provided to `start_recording`.